action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Install dfx'
+description: 'Install the dfx utility at a particular version'
+inputs:
+  dfx-version:
+    description: >
+      The dfx version to install. This value will be used to populate
+      the DFX_VERSION variable that will be handled by the installation
+      script "install.sh".
+    default: 'latest'
```

Comment:
There's a potential disconnect here between:
- the git's notion of `latest`
- the latest promoted dfx version

For example, at the time of this writing, we have:
- git's notion of `latest`: 0.14.2
- latest promoted version: 0.14.1

We intend to make it so these are the same thing, but for the time being, that is not the case.